src/components/bs5/banner/banner.stories.js

```diff
@@ -155,6 +155,17 @@ export const WithCallToAction = {
   },
 }
 
+/**
+ * Banner with Advance Banner
+ */
+export const AdvanceBanner = {
```

Comment:
can you split the story out since vanilla has 3 versions of banner,  (simple, basic, intermediate, advanced)

https://qld-gov-au.github.io/qgds-vanilla/storybook/?path=/docs/components-banner-banner-advanced--overview
https://github.com/qld-gov-au/qgds-vanilla/blob/main/src/components/banner_advanced/stories/bannerAdvanced.stories.js

Banner/Banner (Advanced)